src/wss/v2/trading_messages.rs

```diff
@@ -86,15 +86,18 @@ pub struct AddOrderParams {
     pub cash_order_quantity: Option<Decimal>,
     pub validate: Option<bool>,
     pub token: Token,
+    #[serde(rename = "cl_ord_id")]
+    pub client_order_id: Option<String>,
 }
 
 #[derive(Debug, Deserialize, PartialEq)]
-#[serde(deny_unknown_fields)]
```

Comment:
This was needed in V1 to disambiguate some of the messages due to their formatting, but great it's not needed now. Them adding client order id explains some upstream behavior I saw recently, too.